### DISCUSSION TITLE
change content of a file to a specified hex color? (unsolved)

### DISCUSSION THREAD

**Discussion by payrim** [TriliumNext/Trilium#2076] [Q&A]
1. distinct colored icons with title separation color (**solved**)
2. option to hide relation names in relation maps for each edge
3. new CSS icons or a way to add them **(solved**)
4. maybe adding a drawing note where you could draw simple shapes and notes (basically a dumbed down version of paint, overly simplified)  (**idea**)
5. change content of a file to a specified hex color? (**unsolved**)

**Comment by davialexandre** [comment]
@payrim I'm not sure if this is what you meant by "new CSS icons", but I've been using this technique to be able to use new ones: https://davi.dev/post/emoji-icons-on-trilium-notes/

**Reply by payrim** [reply]
alright... how to use fontawesome?

**Reply by davialexandre** [reply]
I never tried this with fontawesome before, but I did a quick investigation and it looks like the process will be a bit more complicated. The problem is that fontawesome requires some external webfonts, so you'll need to find a way load these too. Maybe you can use the custom request handler approach that is mentioned [here](https://github.com/zadam/trilium/wiki/Themes#custom-css-themes)?

With that done, I believe it should then all be a matter of importing the fontawesome css and applying the ncessary CSS classes

**Comment by Mangiola** [comment]
As for point 1, you can already have colored note icons (assuming I understand the idea). In a CSS note, you'd have a line like this:

 `.fancytree-node.blue .fancytree-custom-icon  { color: blue; }`

Then it's a matter of giving a note the `#cssClass=blue` label. You'd make a bunch more for other colors. I recommend setting the label as inheritable so it makes all children the same color, which I find looks very nice:

![Screenshot_20210722_140013](https://user-images.githubusercontent.com/24983951/126688052-4f91350c-43c1-4282-9e2e-9395f8318a28.png)
